### DISCUSSION TITLE
Typo on Copilot setup incomplete page

### DISCUSSION THREAD

**Discussion by beverlyj93** [community/community#19396] [Copilot Conversations]
![MicrosoftTeams-image](https://user-images.githubusercontent.com/38091445/175750495-1ae997aa-c9d9-4f9a-9506-04beee53e641.png)

The word "blow" is supposed to be "below"

**Comment by y9su** [comment]
Wow I never noticed that! I am pretty sure you can submit this to --> https://support.github.com/contact/bug-report
